cmd/web3/main.go

```diff
@@ -229,6 +229,68 @@ func main() {
 				IncreaseGas(ctx, privateKey, network, c.String("tx"), c.String("amount"))
 			},
 		},
+		{
+			Name:  "replace",
+			Usage: "Replace transaction. If a transaction is still pending, you can attempt to replace it.",
```

Comment:
Does this behave differently than a send/transfer command would with the `nonce` set? Or would adding a `nonce` flag there be equivalent? That would be a lot less API surface and more in line with standard web3 interaction.

Reply:
Hmm, good point. Might be the same thing? Will have to look.

Reply:
Ehh, it's different in a couple of ways, "send"/"transfer" is more english language and only applies to a transfer money. Replace can replace any transaction (although data not supported yet, but should be).